cli.js

```diff
@@ -42,3 +45,13 @@ read(glob(argv._))
     console.error(err);
     process.exit(1);
   });
+
+function errorContext(err, file) {
+  require('colors');
```

Comment:
Just require the dependency up at the top of the file (when you switch to chalk)